qiita_ware/test/test_ebi.py

```diff
@@ -586,13 +587,11 @@ def test_generate_xml_files(self):
         e.generate_demultiplexed_fastq()
         self.assertIsNone(e.run_xml_fp)
         self.assertIsNone(e.experiment_xml_fp)
-        self.assertIsNone(e.sample_xml_fp)
```

Comment:
From this PR I can't see why this attribute is no longer tested. Would you mind explaining?

Reply:
Will re-add, thank you for catching ...